internal/cli/usercmd/client.go

```diff
@@ -105,7 +107,7 @@ func New() (*EpinioClient, error) {
 		return nil, errors.Wrap(err, "error loading settings")
 	}
 
-	apiClient := epinioapi.New(cfg)
+	apiClient := epinioapi.New(context.Background(), cfg)
```

Comment:
And now we need this `New()` function to also accept a context so we can push down the cobra command context. :thinking: